config/fabric.py

```diff
@@ -278,3 +278,66 @@ def recovery(pollcount, namespace):
         {"monPollThreshRecovery": pollcount})
 
 
+#
+# 'config fabric monitor ...'
+#
+@fabric.group(cls=clicommon.AbbreviationGroup, name='monitor')
+def capacity_monitor():
+    """FABRIC MONITOR configuration tasks"""
+    pass
+
+#
+# 'config fabric monitor capacity...'
+#
+@capacity_monitor.group(cls=clicommon.AbbreviationGroup)
+def capacity():
+    """FABRIC MONITOR CAPACITY configuration tasks"""
+    pass
+
+#
+# 'config fabric monitor capacity threshold <capcityThresh>'
+#
+@capacity.command()
+@click.argument('capacitythreshold', metavar='<capacityThreshold>', required=True, type=int)
+def threshold(capacitythreshold):
+    """FABRIC CAPACITY MONITOR THRESHOLD configuration tasks"""
+    ctx = click.get_current_context()
+
+    if capacitythreshold < 5 or capacitythreshold > 250:
+        ctx.fail("threshold must be in range 5...250")
+
+    if multi_asic.is_multi_asic():
```

Comment:
iso of if multiasic check and using `multi_asic.get_all_namespaces()` can we use `multi_asic.get_namespace_list()`. It will return a list for single and multi asic check? Then the code could be common

Reply:
revised